eox_core/api/v1/views.py

```diff
@@ -790,10 +790,10 @@ def get(self, request):
         course_grade = grade_factory().read(user, course)
         response = {"earned_grade": course_grade.percent}
 
-        if detailed:
+        if detailed in ("True", "true", "on", "1"):
```

Comment:
Is this something that we support in the other endpoints?  Does the api docs  says that we can enabled them with "on"?

Reply:
The enrollment endpoint uses a `force` flag and uses a serializer to validate the input. There seems to be [several valid inputs](https://github.com/encode/django-rest-framework/blob/19655edbf782aa1fbdd7f8cd56ff9e0b7786ad3c/rest_framework/fields.py#L705). It also checks for valid false values and this line does not, so any other value will be taken as false. What would be your suggestion?